src/theme/NavbarItem/index.js

```diff
@@ -0,0 +1,54 @@
+/**
```

Comment:
note: this is the same file as upstream, automatically generated by the swizzle command and not changed on our side.